.drone.yml

```diff
@@ -0,0 +1,42 @@
+pipeline:
+   node 9:
+      image: bitgosdk/ubuntu_nvm:18.10
+      secrets: 
+         - password
+      commands: 
+         - export NVM_DIR="/root/.nvm/versions/node/v9.11.2/bin" 
+         - export PATH=$NVM_DIR:$PATH
+         - export BITGOJS_TEST_PASSWORD=$PASSWORD
+         - node --version
```

Comment:
I don't think we need to print the node version, given that we are defining it two lines above

Reply:
I had suggested to execute `node --version` and `npm --version` for debugging purpose as Perry had run into issue earlier.